### ISSUE TITLE
Deploy Pavise skill description wrong?

### ISSUE THREAD

**Issue by Soukyuu** [Treeofsavior/EnglishTranslation#819]
So the description says:

```
SKILL_20150317_001306   Install Pavise on ground and defend long rage projectile.
SKILL_20150317_001307   Defend range projectile #{SkillFactor}# times{nl}Pavise duration #{CaptionTime}# seconds
```

Typos aside, I've tested it ingame and it also defends you against melee attacks. There is also the fact deploying it actually deals damage. Is the description wrong, or is that a bug I should report on the forums?

**Comment by Sunaries** [comment]
I've translated the skill description from Korean to English for you:

SKILL_20150317_001306 지면에 파비스를 설치해 원거리 투사체를 방어합니다.
SKILL_20150317_001306 Set up/Put a Pavise on the ground to defend yourself from projectiles. 

SKILL_20150317_001307 원거리 투사체 {SkillFactor}회 방어 {nl} 파비스 지속시간 {CaptionTime}초
SKILL_20150317_001307 Defends a projectile (SkillFactor) times {nl} Pavise Duration {CaptionTime}seconds

I would report it on the forums, unless they consider a melee attack a projectile. I doubt that.

**Comment by Soukyuu** [comment]
Just from looking at the way the skill works, it would actually make sense for it to block melee attacks too - if the pavise is between you and your enemy, it acts like a shield. Maybe the skill description is incomplete in Korean? I'll report it anyway.

**Comment by Sunaries** [comment]
> Just from looking at the way the skill works, it would actually make sense for it to block melee attacks too - if the pavise is between you and your enemy, it acts like a shield. Maybe the skill description is incomplete in Korean? I'll report it anyway.

Yeah, it works like that exactly. It would be better to rephrase it to: "_Set up/Put a Pavise on the ground to defend yourself from incoming attacks._" - But it says projectile in the description.

**Comment by Soukyuu** [comment]
I'm sure @imcgames will see my bug report on the forums, but I think it might be faster to just alert them here.

**Comment by Kuruadeo** [comment]
I've noticed that the Pavise can draw aggro from monsters and can be attacked directly by monsters. I'm not sure how monster attacks function in terms of melee. 
Perhaps they have a max hit capacity of 1 (whereas a player Swordsman can hit 3 in one swing), but it impacts the Pavise first, negating its attack? That's my best guess, but I'm unable to personally test it in-game as the beta ended. Whether it's a bug or intentional we'll need to wait and see
